apps/src/aiComponentLibrary/copyButton/CopyButton.tsx

```diff
@@ -0,0 +1,46 @@
+import Button from '@code-dot-org/component-library/button';
+import React, {useState} from 'react';
+
+import style from './copy-button.module.scss';
+
+// Fallback method for browsers that do not support navigator.clipboard
+const copyToClipboard = (text: string) => {
+  const textField = document.createElement('textarea');
+  textField.innerText = text;
+  document.body.appendChild(textField);
+  textField.select();
+  document.execCommand('copy');
+  textField.remove();
+};
+
+const CopyButton: React.FC<{copyText: string}> = ({copyText}) => {
+  const CONFIRM_TIMEOUT_MS = 1500;
+  const [copyTimeout, setCopyTimeout] = useState(false);
```

Comment:
nit: this sort of feels like it's storing a timeout value or the timeout callback ID - maybe `showCopyTimeout` or `showCopyConfirmation`?

Reply:
That makes sense to me, will update!